src/components/BloombergToFDC3/common.ts

```diff
@@ -0,0 +1,8 @@
+export const LINK_PREFERENCES_PATH = ["finsemble", "custom", "bloomberg", "links"];
+
+
+export const getDisplayName = (link) => {
+  const tailsPart = link.target.args?.tails ? ` ${link.target.args.tails}` : "";

```

Comment:
Personally, I prefer this pattern, but that's just me:

```suggestion
  const tailsPart = link.target.args?.tails ?? ` ${link.target.args.tails}`;

```

Reply:
Thats not achieving the same thing - we had to switch from what you propose. We want an empty string with no space if its tails isn't set